bitnami/kube-prometheus/values.yaml

```
@@ -3307,3 +3307,565 @@ kubeProxy:
+  ## ref: https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#inter-pod-affinity-and-anti-affinity
+  ##
+  podAffinityPreset: ""
+  ## @param thanosRuler.podAntiAffinityPreset Thanos Ruler Pod anti-affinity preset. Ignored if `affinity` is set. Allowed values: `soft` or `hard`
+  ## ref: https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#inter-pod-affinity-and-anti-affinity
+  ##
+  podAntiAffinityPreset: soft
+  ## Node affinity preset
+  ## ref: https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#node-affinity
+  ##
+  nodeAffinityPreset:
+    ## @param thanosRuler.nodeAffinityPreset.type Thanos Ruler Node affinity preset type. Ignored if `affinity` is set. Allowed values: `soft` or `hard`
+    ##
+    type: ""
+    ## @param thanosRuler.nodeAffinityPreset.key Thanos Ruler Node label key to match. Ignored if `affinity` is set
+    ## E.g.
+    ## key: "kubernetes.io/e2e-az-name"
+    ##
+    key: ""
+    ## @param thanosRuler.nodeAffinityPreset.values Thanos Ruler Node label values to match. Ignored if `affinity` is set
+    ## E.g.
+    ## values:
+    ##   - e2e-az1
+    ##   - e2e-az2
+    ##
+    values: []
+  ## @param thanosRuler.affinity Thanos Ruler Affinity for pod assignment
+  ## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
+  ## Note: thanosRuler.podAffinityPreset, thanosRuler.podAntiAffinityPreset, and thanosRuler.nodeAffinityPreset will be ignored when it's set
+  ##
+  affinity: {}
+  ## @param thanosRuler.podMetadata [object] Standard object's metadata
+  ## ref: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#metadata
+  ##
+  podMetadata:
+    labels: {}
+    annotations: {}
+  ## @param thanosRuler.tolerations Thanos Ruler Tolerations for pod assignment
+  ## ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
+  ##
+  tolerations: []
+  ## @param thanosRuler.topologySpreadConstraints Topology Spread Constraints for pod assignment
+  ## https://kubernetes.io/docs/concepts/workloads/pods/pod-topology-spread-constraints/
+  ## The value is evaluated as a template
+  ##
+  topologySpreadConstraints: []
+  ## @param thanosRuler.containers Containers allows injecting additional containers or modifying operator generated containers
+  ## The current container names are: thanos-ruler and config-reloader
+  ##
+  containers: []
+  ## @param thanosRuler.initContainers InitContainers allows adding initContainers to the pod definition
+  ## Those can be used to e.g. fetch secrets for injection into the ThanosRuler configuration from external sources.
+  ##
+  initContainers: []
+  ## @param thanosRuler.priorityClassName Priority class assigned to the Pods
+  ##
+  priorityClassName: ""
+  ## @param thanosRuler.portName Port name used for the pods and governing service. Defaults to web
+  ##
+  portName: "web"
```

Default value not required in parameter description
```suggestion
  ## @param thanosRuler.portName Port name used for the pods and governing service
  ##
  portName: web
```